cmd/vsphere-xcopy-volume-populator/vmkfstools-wrapper/vmkfstools_wrapper.py

```
@@ -122,6 +134,32 @@ def extract_rdmdisk_file(rdm_file):
         logging.error(e)
     return ""
 
+def was_xcopy_used(target_lun):
```

Ruff (0.12.2)

Missing return type annotation for public function `was_xcopy_used`

(ANN201)